package.json

```
@@ -1,5 +1,5 @@
 {
-    "name": "accounts",
+    "name": "hub",
```

I know it's not published, but still.
```suggestion
    "name": "@nimiq/hub",
```